CMSIS/Core_R/Include/cmsis_compiler.h

```diff
@@ -0,0 +1,213 @@
+/**************************************************************************//**
+ * @file     cmsis_compiler.h
+ * @brief    CMSIS compiler specific macros, functions, instructions
+ * @version  V1.0.2
+ * @date     10. January 2018
```

Comment:
We should set proper version and date information. I guess the file is simply a copy. But nevertheless the Cortex-R variant should be seen as a new file, i.e. starting at version 1.0.0